Dockerfile.template

```diff
@@ -19,10 +20,12 @@ RUN apt-get update && \
         python-simplejson \
         libraspberrypi0 \
         ifupdown \
+        python-urllib3 \
         sqlite3 \
         uzbl \
         x11-xserver-utils \
         xserver-xorg && \
+    apt full-upgrade \
```

Comment:
This goes against Docker best-practices, which states that you rely on the upstream image for this.